test/e2e/main_test.go

```diff
@@ -134,6 +135,15 @@ func TestTargetsUp(t *testing.T) {
 		)
 	}
 
+	// Make sure CMO metrics are up.
+	f.ThanosQuerierClient.WaitForQueryReturnOne(
+		t, 5*time.Minute, "cluster_monitoring_operator_last_reconciliation_successful",
+	)
+
+	f.ThanosQuerierClient.WaitForQueryReturnGreaterEqualOne(
+		t, 5*time.Minute, "cluster_monitoring_operator_reconcile_attempts_total",
+	)
+
```

Comment:
this isn't needed, what we want to ensure is that CMO is scraped successfully by Promehteus and the query on the up metric already checks this. 

```suggestion
```